packages/block-library/src/cover/edit.native.js

```diff
@@ -305,7 +341,16 @@ const Cover = ( {
 					onSelect={ onSelectMedia }
 					allowedTypes={ ALLOWED_MEDIA_TYPES }
 					onFocus={ onFocus }
-				/>
+				>
+					<ColorPalette
+						customStyles={ styles }
+						setColor={ setColor }
+						onCustomPress={ () => {} }
```

Comment:
Here instead of creating an arrow function, we could pass `noop` from `import { noop } from 'lodash';`